fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/RedirectHandlerService.java

```diff
@@ -103,7 +106,11 @@ private ResponseEntity prepareErrorRedirectResponse(String redirectUrl) {
     }
 
     private String getModifiedUrlWithRedirectCode(String... params) {
-        return UriComponentsBuilder.fromPath(redirectUrl)
+        return UriComponentsBuilder.newInstance()
```

Comment:
I think if you use 
```
redirect-url: http://localhost:4444/{redirectUri}/{redirectCode}
```
(notice the slash)
then 
UriComponentsBuilder.fromHttpUrl can yield simpler code like this:
```
UriComponentsBuilder.fromHttpUrl("http://localhost:4444/{redirectUri}/{redirectCode}")
        .buildAndExpand(ImmutableMap.of('redirectUri', "path", 'redirectCode', "12345"))
        .toUriString();
```

```
Result: http://localhost:4444/path/12345
```